src/tasks/start/frontend/aragon-client.ts

```diff
@@ -54,13 +53,13 @@ export async function installAragonClientIfNeeded(
 async function _checkClientInstallationNeeded(
   clientPath: string
 ): Promise<boolean> {
-  if (!fs.existsSync(path.resolve(clientPath))) {
+  if (!pathExists(clientPath)) {
     return true
   }
 
-  if (!fs.existsSync(path.resolve(clientPath, 'build/index.html'))) {
+  if (!pathExists(path.resolve(clientPath, 'build/index.html'))) {
     logFront('Malformed client detected, removing it for re-installation.')
-    await fsExtra.remove(clientPath)
+    remove(clientPath)
```

Comment:
Sanity checking, but should we do an `await` on the `remove()`? What if this fails—do we handle the promise ever rejecting?